mysql-test/suite/funcs_1/r/processlist_priv_ps.result

```diff
@@ -31,31 +31,34 @@ PROCESSLIST	CREATE TEMPORARY TABLE `PROCESSLIST` (
   `COMMAND` varchar(16) NOT NULL DEFAULT '',
   `TIME` int NOT NULL DEFAULT '0',
   `STATE` varchar(64) DEFAULT NULL,
-  `INFO` longtext
```

Comment:
It looks like `funcs_1.processlist_val_ps` has the same problem. I would include it into this commit as well.

Reply:
Submitted updated version of patch.